configuration-reference/functions/firstwithtime.md

```diff
@@ -0,0 +1,24 @@
+---
+description: This section contains reference documentation for the firstwithtime function.
+---
+
+# firstwithtime
+
+Get the first value of `dataColumn` where the `timeColumn` is used to define the time of `dataColumn` and the `dataType` specifies the type of `dataColumn`, which can be `BOOLEAN`, `INT`, `LONG`, `FLOAT`, `DOUBLE`, `STRING`
```

Comment:
Looking at the [code](https://github.com/apache/pinot/blob/e08e66c3991b40f0d95c48b9436b97d406fa3311/pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/FirstWithTimeAggregationFunction.java#L37C1-L47C4) first two arguments to the function are expressions and third is a literal specifying the return type of dataColumn. 
Help snippet from code:
`/**
 * This function is used for FirstWithTime calculations.
 * <p>The function can be used as FirstWithTime(dataExpression, timeExpression, 'dataType')
 * <p>Following arguments are supported:
 * <ul>
 *   <li>dataExpression: expression that contains the column to be calculated first on</li>
 *   <li>timeExpression: expression that contains the column to be used to decide which data is first, can be any
 *   Numeric column</li>
 *   <li>dataType: the data type of data column</li>
 * </ul>
 */`
 
 This seems more easier to understand definition vs current one.

Reply:
`timeColumn` can be of type `TIMESTAMP`, `INT`, `LONG`